test/correctness/simplify.cpp

```diff
@@ -1713,6 +1713,7 @@ void check_boolean() {
     check(select(x == 3, 5, y) - y, select(x == 3, 5 - y, 0));
     check(select(x == 3, y, 5) - y, select(x == 3, 0, 5 - y));
     check(y - select(x == 3, 5, y), select(x == 3, y + (-5), 0));
+    check(y - select(x == 3, 5, y), select(x == 3, y, 5) + (-5));
```

Comment:
I think you intended to replace the line above this with this one?

Reply:
Yes my bad..